### DISCUSSION TITLE
Is it optimal to run it on Linux or Windows?

### DISCUSSION THREAD

**Discussion by Albaricoque695** [hacksider/Deep-Live-Cam#825] [Q&A] [answered]
That's my question

**Comment by KRSHH** [comment, marked as answer]
Windows.

**Reply by Albaricoque695** [reply]
11 or 10?

**Reply by KRSHH** [reply]
tested Mostly on 11, but, should work completely same in 10.

**Reply by TouchFotos** [reply]
I'm attempting to set it up on my own laptop, but I'm encountering issues during the pip install -r requirements.txt phase. I'm not sure what the problem might be, so I'm reaching out to you for assistance.

I believe there might be some specific requirements or steps that I'm overlooking.

My system specifications are as follows:

Laptop: HP Laptop 14-dq2xxx
Processor: 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40 GHz
Graphics: Intel(R) Iris(R) Xe Graphics
RAM: 8GB

Please let me know if you think Deep Live Cam is compatible with my system and if you're available to provide some guidance on the setup process from scratch. By the way, when will the CPU version come out?
